app/views/instance_actors/show.html.haml

```diff
@@ -0,0 +1,48 @@
+- content_for :page_title do
```

Comment:
I can't really tell how this is going to look from code alone but do we *need* a HTML page for this?

Reply:
The page is useful to let people know what this actor is about, as it may be confusing. Having it on a separate page allows people to know at a glance that it is a special user.

It looks like this:
![image](https://user-images.githubusercontent.com/384364/61241511-ed749a00-a743-11e9-876a-da7dd96f00e0.png)

Reply:
Should the `url` maybe just point to the `/about` page then? What I'm trying to avoid is the maintenance cost of another static page design because they've tended to be redesigned quite a few times so far.

Reply:
I don't think so, we really want to explain what this account is for. I should probably reword that description btw.